README.md

```diff
@@ -150,6 +150,32 @@ If you want to deploy the infrastructure yourself from template you can also cli
 
 [![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fmicrosoft%2Frag-experiment-accelerator%2Fdevelopment%2Finfra%2Fmain.json)
 
+### 4. Deploy with Make command (with option to deploy in isolated network)
```

Comment:
Can't the network isolation be added into one of the other options?
Why do we need to use `Makefile` when `azd` is used already?

If we can avoid adding another way to deploy stuff, that would be better.

Reply:
I understand why this has happened
So both of these options were implemented at (relatively) same time.
And CLI way of deployment has issue (which renders it not usable atm) [#486](https://github.com/microsoft/rag-experiment-accelerator/issues/486)
There are 2 options I can resolve this comment 

Options 1:

Resolve the issue #486  [in CLI way of deployment]
And remove Options of deployment from Makefile and only have CLI 

Option 2:

Remove CLI way of deployment

which one do you think is appropriate here?

Reply:
Issue #486 looks easy to resolve. The command line example in the README was never updated when @ross-p-smith added `targetScope = 'subscription'` to the main.bicep file. So, the command line sample just needs updating.

I would remove (or, rather, "not add") the Makefile option.

Reply:
Resolved #486 with #506

Reply:
Thanks :)